stock_orderpoint_manual_procurement/wizards/make_procurement_orderpoint.py

```diff
@@ -69,6 +69,7 @@ def make_procurement(self):
             values = item.orderpoint_id._prepare_procurement_values(item.qty)
             values['date_planned'] = fields.Datetime.to_string(
                 fields.Date.from_string(item.date_planned))
+            values['requested_uid'] = self.env.user
```

Comment:
@ageficent @lreficent  Is that field introduced through an OCA module ?

Reply:
it is not a field itself, the purpose is to have available the information of who have made the procurement. This can be used in the destination model (see for MO: https://github.com/OCA/manufacture/pull/328).

Reply:
Ok, wasn't clear